test/xpu_api/algorithms/alg.sorting/alg.sort/is.sorted/xpu_is_sorted_until.pass.cpp

```diff
@@ -0,0 +1,90 @@
+//===-- xpu_is_sorted_until.pass.cpp --------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/algorithm>
+
+#include "support/test_iterators.h"
+
+#include <cassert>
+#include <CL/sycl.hpp>
+
+template <class Iter>
+void
+test(sycl::queue& deviceQueue)
+{
+    using T = typename std::iterator_traits<Iter>::value_type;
+    bool ret = true;
+    sycl::range<1> item1{1};
+    {
+        sycl::buffer<bool, 1> buffer1(&ret, item1);
+        deviceQueue.submit([&](sycl::handler& cgh) {
+            auto ret_acc = buffer1.get_access<sycl::access::mode::write>(cgh);
+            cgh.single_task<Iter>([=]() {
+                {
+                    T a[] = {1, 0};
+                    unsigned sa = sizeof(a) / sizeof(a[0]);
+                    ret_acc[0] &= (std::is_sorted_until(Iter(a), Iter(a + sa)) == Iter(a + 1));
```

Comment:
I understand that we have an injection the names from `std ` namespace to `dpl `namespace.
Everywhere in the `xpu` tests we use `dpl::` namespace for the tested  serial algorithms, but here(and below) `std `- it looks inconsistently, I mean `std::is_sorted_until `instead of  `dpl::is_sorted_until`.

Reply:
Oh, sorry, `std::is_sorted_until` should be replaced with `dpl::is_sorted_until`.

Reply:
Fixed